src/globus_cli/parsing/commands.py

```diff
@@ -74,7 +75,28 @@ def parse_args(self, ctx: click.Context, args: list[str]) -> list[str]:
         # args will be consumed, so check it before super()
         had_args = bool(args)
         try:
+            # if we have any opts to be combined in order, do that now
+            if self.opts_to_combine:
+                combined_opts: dict[str, list[tuple[str, str]]] = {
+                    combined_name: [] for combined_name in self.opts_to_combine.values()
+                }
+                parser = self.make_parser(ctx)
+                values, _, order = parser.parse_args(args=list(args))
```

Comment:
I think it would be valuable to note what the type of `parser` is here. I think it's an optparse parser?

Reply:
> Before merging, I'd like to see if we can add a comment to describe the internal parser object a little. It will make it easier to come back to this code in the future. That said, we can do without that if we want to release this shortly.

@sirosen I added some explanation and example to the `GlobusCommand` help text, not sure that's exactly what you wanted?